grakn-graql/src/test/java/ai/grakn/graql/internal/query/DeleteQueryTest.java

```diff
@@ -150,42 +150,42 @@ public void testDeleteSpecificName() {
                         .has("gender", "male")
         ).execute();
 
-        assertTrue(qb.match(var().isa("person").has("real-name", "Bob")).ask().execute());
-        assertTrue(qb.match(var().isa("person").has("real-name", "Robert")).ask().execute());
-        assertTrue(qb.match(var().isa("person").has("gender", "male")).ask().execute());
+        assertTrue(qb.match(var().isa("person").has("real-name", "Bob")).iterator().hasNext());
```

Comment:
Possible Nitpick: For these guys it might be better to use Iterables.getOnlyElement(...). . . . assuming only one is supposed to be returned.